src/main/java/edu/hawaii/its/api/util/JsonUtil.java

```diff
@@ -29,7 +35,17 @@ public static <T> T asObject(final String json, Class<T> type) {
             result = new ObjectMapper().readValue(json, type);
         } catch (Exception e) {
             logger.error("Error: " + e);
-            // Maybe we should throw something?
+        }
+        return result;
+    }
+
+    public static <T> List<T> asObjectList(final String json, Class<T> type) {
```

Comment:
I think this function should be renamed to asList.

Reply:
I've changed the function name in the last commit, thank you.